addons/mass_mailing/models/mailing.py

```diff
@@ -488,8 +498,19 @@ def action_test(self):
         }
 
     def action_launch(self):
-        self.write({'schedule_type': 'now'})
-        return self.action_put_in_queue()
+        mailing_send_immediate_nbr = self.env['ir.config_parameter'].sudo().get_param('mass_mailing.mass_mailing_immediate_send_nbr', 20)
```

Comment:
Would not give 20 as default value. If we have a data config parameter, default would be 0 and "skip auto sending".